ogc-example/src/main/scala/geotrellis/server/ogc/conf/RasterSourceConf.scala

```diff
@@ -25,7 +25,7 @@ import geotrellis.store.{GeoTrellisPath, GeoTrellisRasterSourceLegacy}
  *  a geotrellis-contrib [[RasterSource]].
  */
 sealed trait RasterSourceConf {
-  def toRasterSource: RasterSource
+  def toRasterSources: List[RasterSource]
```

Comment:
This seems like a significant alteration, but I couldn't figure out what it was for (the only implementation I saw here just attached the lone RS to `Nil`)

Reply:
It was to make use of the Repository interface. The whole thing was to make queriable temporal layers and a temporal layer is just a set of RasterSources. So having a List of RasterSources read from the configuration can allow as to create an instance of a Repository that we can query by time as well.